src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.After.targets

```diff
@@ -0,0 +1,20 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Bindings.Before.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+  created a backup copy.  Incorrect changes to this file will make it
+  impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) 2014 Xamarin. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Allows providing SDK-specific targets overrides -->
+  <PropertyGroup>
+    <XamarinAndroidSdkTargets>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Xamarin.Android.Sdk.targets'))\Xamarin.Android.Sdk.targets</XamarinAndroidSdkTargets>
```

Comment:
Because this way I can move that file up or down freely ;), and I can just copy/paste those two lines regardless of which file I'm in. (or if it moves its own location elsewhere in the future)